src/platform/user/profile/vap-svc/components/PhoneField/PhoneField.jsx

```diff
@@ -34,37 +33,29 @@ const formSchema = {
   required: ['inputPhoneNumber'],
 };
 
-const uiSchema = {
-  'view:noInternationalNumbers': {
-    'ui:description': () => (
-      <AlertBox isVisible status="info" className="vads-u-margin-bottom--3">
-        <p>
-          We can only support U.S. phone numbers right now. If you have an
-          international number, please check back later.
-        </p>
-      </AlertBox>
-    ),
-  },
-  inputPhoneNumber: {
-    'ui:widget': PhoneNumberWidget,
-    'ui:title': 'Number',
-    'ui:errorMessages': {
-      pattern: 'Please enter a valid phone number.',
+const uiSchema = fieldName => {
```

Comment:
This component is not used on the profile. It is used by the `WorkPhone`, `MobilePhone`, `HomePhone` and `FaxNumber` components, which are also not used on the profile, but I vaguely remember having a conversation about not removing these components because other apps might want to use them in the future. If this is no longer a wish, I would be happy to remove this currently unused code.